vault/data_source_kubernetes_auth_backend_role_test.go

```diff
@@ -54,19 +55,19 @@ func TestAccKubernetesAuthBackendRoleDataSource_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("data.vault_kubernetes_auth_backend_role.role",
 						"role_name", role),
 					resource.TestCheckResourceAttr("data.vault_kubernetes_auth_backend_role.role",
-						"bound_service_account_names.0", "example"),
+						"bound_service_account_names.64447719", "example"),
 					resource.TestCheckResourceAttr("data.vault_kubernetes_auth_backend_role.role",
 						"bound_service_account_names.#", "1"),
 					resource.TestCheckResourceAttr("data.vault_kubernetes_auth_backend_role.role",
-						"bound_service_account_namespaces.0", "example"),
+						"bound_service_account_namespaces.64447719", "example"),
```

Comment:
Why did these numbers change in this manner throughout this file?

Reply:
I changed the type of these fields from `schema.TypeList` to `schema.TypeSet`. So in the state, they are now indexed by some hash rather than their indices.